src/backend/src/migrations/1669227797131-pia-intake-entity.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class PiaIntakeEntity1669227797131 implements MigrationInterface {
+  name = 'piaIntakeEntity1669227797131';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `CREATE TABLE "pia-intake" ("id" SERIAL NOT NULL, "is_active" boolean NOT NULL DEFAULT true, "created_at" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(), "updated_at" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(), "title" character varying NOT NULL, "ministry" character varying NOT NULL, "branch" character varying NOT NULL, "drafter_info" character varying NOT NULL, "drafter_email" character varying NOT NULL, "lead_info" character varying NOT NULL, "lead_email" character varying NOT NULL, "mpo_info" character varying NOT NULL, "mpo_email" character varying NOT NULL, "initiative_description" character varying NOT NULL, "initiative_scope" character varying NOT NULL, "data_elements_involved" character varying NOT NULL, "has_added_pi_to_data_elements" boolean, "risk_mitigation" character varying, CONSTRAINT "PK_11a96bc6c11cdc346816707c095" PRIMARY KEY ("id"))`,
```

Comment:
Is there a easier way to break this down. Wondering if someone makes some alterations to this SQL statement it might completely fail to execute. Easy to break this.

Reply:
If there is a method you are using to test this statement add it in the comments. It can help other users too to test if the statement works.

Reply:
Ideally, no one should make any changes to a migration scripts. Once run, they should not be touched. If any alterations are required, a new migration should be written on top of it.

The method to test is by allowing migration to run (when app starts or by manually running `npm run migration:run`) and validating the table schema generated against the entity in the codebase.

Reply:
Perfect! Can you add a comment to the top of this file based on what you stated? Every migration script can thus follow a coding guideline:

Description / Changes made:
How to test:

Reply:
Added Migration summary to the file. However, testing a migration is somewhat common and should be in some sort of common place, a wiki or md or even github template. What do you think?

Reply:
Agree. This is more for new/fresh developers to navigate the code base with minimum effort.